pkg/networkservice/chains/nsmgr/server_test.go

```diff
@@ -65,19 +67,13 @@ func newClient(ctx context.Context, u *url.URL) (*grpc.ClientConn, error) {
 func TestNSmgrEndpointCallback(t *testing.T) {
 	grpclog.SetLoggerV2(grpclog.NewLoggerV2(os.Stdout, os.Stdout, os.Stderr))
 
-	ctx, cancel := context.WithTimeout(context.Background(), 150*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 1500*time.Second)
```

Comment:
[minor]
```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), 25*time.Minute)
```

Reply:
Good catch :-)

Reply:
It will be changed in cross NSE PR, so will fix it inside it.